build/configure

```diff
@@ -94,6 +94,9 @@ case $uname in
 	    ppc64*)
 		PF_ARCH=ppc
 		ARCH=ppc
+		VERSION=`grep 'Red Hat Enterprise' /etc/redhat-release | sed s/.*release\ // | sed s/\ .*//`
+		PF_MAJOR=`echo $VERSION | sed s/[.][0-9]*\//`
+	        PF_MINOR=`echo $VERSION | sed s/\[0-9]*[.]//`
```

Comment:
These lines are using a mix of tab and space indent, please just use space indentation here.